frontend/components/text-animation.tsx

```diff
@@ -3,6 +3,7 @@
 import { motion, useMotionValue, useTransform, animate } from "framer-motion";
 import { useEffect } from "react";
 import BlinkingCursor from "./blinking-cursor";
+import React from "react";
```

Comment:
Isn't it better to just import the function you're using from react instead of the whole object?

Reply:
Not really. Just a matter of style tbh. Next.js supports dependency tree shaking (removing unused deps + code) natively.

I just like to use React namespace because it's just descriptive. e.g. `React.PropsWithChildren`. I prefer that instead of just `PropsWithChildren`.

Also, when I'm looking at a component, seeing things e.g. `React.useState` just helps me see pretty quickly which hooks are from React.

Just a matter of style.

Reply:
snm bro